internal/management/repository/eventsourcing/repository.go

```diff
@@ -24,6 +25,8 @@ type Config struct {
 
 type EsRepository struct {
 	spooler *es_spol.Spooler
+	eventstore.OrgRepository
+	eventstore.OrgMemberRepository
```

Comment:
Should we do Repos by Object or by Aggregate (Org is by object, rest is by aggregate)

Reply:
did it analog project by aggregate